### PR TITLE
adding webcomponent packages, tooling and samples

### PR DESCRIPTION
**Pull request by lemaiwo**

### CODE REVIEW

File: sources.json [hunk 1]
```diff
@@ -647,5 +647,36 @@
...
+    "addedToBoUI5": "2023-08-25T00:00:00.007Z",
+    "type": "module",
+    "tags": ["tooling","webcomponents","generator"],
+    "license": "apache-2.0"
+  },
+  {
+    "owner": "lemaiwo",
+    "repo": "ui5-space-webcomponent-library",
+    "addedToBoUI5": "2023-08-25T00:00:00.007Z",
+    "type": "application",
```
Comment (marianfoo) [inline sources.json:670]: if you want your application to be visible on this site
https://bestofui5.org/#/applications
you need to add this to the website itself, its not coming from the data
https://github.com/ui5-community/bestofui5-website/blob/main/src/view/Applications.view.xml

File: sources.json [hunk 2]
```diff
@@ -647,5 +647,36 @@
...
+    "owner": "SAP",
+    "repo": "ui5-tooling-webc",
+    "addedToBoUI5": "2023-08-25T00:00:00.007Z",
+    "type": "module",
+    "tags": ["tooling","webcomponents","generator"],
+    "license": "apache-2.0"
+  },
+  {
+    "owner": "lemaiwo",
+    "repo": "ui5-space-webcomponent-library",
```
Comment (marianfoo) [inline sources.json:668]: i need in the root repo a file called `README.md` and `LICENSE`

File: sources.json [hunk 3]
```diff
@@ -647,5 +647,36 @@
...
+    "addedToBoUI5": "2023-08-25T00:00:00.007Z",
+    "type": "application",
+    "tags": ["sample", "webcomponents", "UI5con2023", "library"],
+    "license": "apache-2.0"
+  },
+  {
+    "owner": "lemaiwo",
+    "repo": "ui5-space-webcomponent-package",
```
Comment (marianfoo) [inline sources.json:676]: same here, i need in the root repo a file called `README.md` and `LICENSE`

**Review by marianfoo — CHANGES_REQUESTED** [review]

File: sources.json (hunk 1 above)
Comment (lemaiwo) [reply sources.json:670]: Thanks! Not sure if that's needed as this is rather a sample library and not really an application

File: sources.json (hunk 2 above)
Comment (lemaiwo) [reply sources.json:668]: License added to the repo

File: sources.json (hunk 3 above)
Comment (lemaiwo) [reply sources.json:676]: License added to the repo

File: sources.json (hunk 1 above)
Comment (marianfoo) [reply sources.json:670]: Sure, justed wanted to point it out if you want to add it

File: sources.json (hunk 2 above)
Comment (marianfoo) [reply sources.json:668]: Sorry, do you mind renaming the readme to `README.md`
It seems this is the standard formatting for the file and my code is looking for this file format

Comment (lemaiwo) [reply sources.json:668]: Done!

File: sources.json (hunk 3 above)
Comment (lemaiwo) [reply sources.json:676]: Done!

**Review by marianfoo — APPROVED** [review]

**Pull request merged**